tasks/retrieval_deal.go

```diff
@@ -260,8 +259,31 @@ func (de *retrievalDealExecutor) executeAndMonitorDeal(ctx context.Context, upda
 				}
 
 				if de.task.CARExport.x {
-					return errors.New("car export not implemented")
+					carPath := filepath.Join(de.config.NodeDataDir, de.task.PayloadCID.x+".car")
```

Comment:
@willscott I believe https://github.com/filecoin-project/dealbot/pull/235/files#diff-6714f1aa5772e99d5e914178bb06c11557a44ff9d6934cfbad3ff02e1dcffd39R160 will actually make LOTUS generate a car for this retrieval automatically. This code was originally meant to double check the result I guess by scanning the CAR file? I'm not sure. I think you can just remove this if statement entirely.